README.md

```diff
@@ -73,9 +73,9 @@ For IntelliJ:
 
 Interested in working for Improbable on cool problems? Start [here](https://improbable.io/careers/joining-us)
 
-[travis-image]: https://travis-ci.org/improbable-research/keanu.svg?branch=develop
+[travis-image]: https://api.travis-ci.org/improbable-research/keanu.svg?branch=develop
 [travis-url]: https://travis-ci.org/improbable-research/keanu
-[maven-image]: https://maven-badges.herokuapp.com/maven-central/io.improbable/keanu/badge.svg
-[maven-url]: https://maven-badges.herokuapp.com/maven-central/io.improbable/keanu
+[maven-image]: https://img.shields.io/maven-central/v/io.improbable/keanu.svg?colorB=brightgreen
+[maven-url]: https://search.maven.org/artifact/io.improbable/keanu/0.0.12/jar
```

Comment:
Do we have to manually change this for every version bump?

Reply:
Looks like just `https://search.maven.org/artifact/io.improbable/keanu/`takes you to the latest version

Reply:
Sorry - yeah, I'll fix it.